proposals/lambda-method-group-defaults.md

```diff
@@ -145,31 +190,44 @@ only been supported since C# 10, so only code which has been written since then
 ### Grammar and Parser Changes
 This enhancement requires the following changes to the grammar for lambda expressions.
 ```diff
-explicit_anonymous_function_parameter
+ explicit_anonymous_function_parameter_list
++    : explicit_anonymous_function_parameters
++    | explicit_anonymous_function_parameters ',' parameter_array
++    | parameter_array
++    ;
++
++explicit_anonymous_function_parameters
+     : explicit_anonymous_function_parameter
+       (',' explicit_anonymous_function_parameter)*
+     ;
```

Comment:
Looking at the baseline spec for [method parameters](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/classes.md#1462-method-parameters), consider whether some parts should be repeated here.

For instance, "A parameter with a ref, out or this modifier cannot have a default_argument.", "A parameter_array may occur after an optional parameter, but cannot have a default value – the omission of arguments for a parameter_array would instead result in the creation of an empty array."

Or we could just refer to that section and say that the same rules apply.

Reply:
Makes sense, thanks. 3649b5bf0a6060442c998d8c46167a23504d65b8